site/_posts/2018-11-19-r-spark-improvements.md

```diff
@@ -182,15 +182,17 @@ Unit: seconds
 </div>
 
 Additional benchmarks and fine-tuning parameters can be found under `sparklyr`
-[/rstudio/sparklyr/pull/1611][9]. Looking forward to bringing this feature
+[/rstudio/sparklyr/pull/1611][10] and `sparkR` [/apache/spark/pull/22954][11]. Looking forward to bringing this feature
```

Comment:
we usually use `SparkR` capitalization...

Reply:
Fixing